### PR TITLE
qualify record model externalId (#137)

### PR DESCRIPTION
**Pull request by tomkralidis**
Addresses #137

### CODE REVIEW

**Review by pvretano — APPROVED** [review]

**Comment by cportele** [comment]
@tomkralidis - The externalId query parameter values are compared against the "value" member and "scheme" is ignored, correct?

**Comment by pvretano** [comment]
@cportele Yes, and I will add that clarification to the document.

**Comment by pvretano** [comment]
@tomkralidis the descriptive text for the value of the external identifier reads:
```
An identifier for the knowledge organization system used to classify the resource.  It is recommended that the
identifier be a resolvable URI
```
I would reword this slightly to not use the term "classify the resource".  That terminology bumps into the purpose of the `themes` core queryable.  Rather, the text should say something like:
```
A reference to an authority or identifier for a knowledge organization system from which the external identifier was obtained.  It is recommended that the identifier be a resolvable URI.
```
If you agree, can you update the text directly in the master branch?  Sorry about that, I should have caught that before merging the PR.

**Comment by tomkralidis** [comment]
@pvretano done in #143
